currencies/rate_converter.go

```diff
@@ -160,14 +168,25 @@ func (rc *RateConverter) LastUpdated() time.Time {
 
 // Rates returns current conversions rates
 func (rc *RateConverter) Rates() Conversions {
```

Comment:
Ok, cool. To verify, it's not impossible to get a nil response from Rates(), right?